saleor/graphql/product/tests/test_product.py

```diff
@@ -2242,6 +2242,73 @@ def test_create_product(
     assert color_value_slug in values
 
 
+def test_create_search_product_by_description(
```

Comment:
We have separate tests for product creation so IMO this test should be strictly about the search. If someone breaks product creation and run the tests, this one would also fail (because it's going to create mutation) and we'd have to wonder why the search broke 🤔 I would suggest to use the fixture and modify it to our needs / create new one if needed.

Reply:
To work properly `SearchVector` needs to be updated after create/update product also we have logic for update `SearchVector` in update/create product mutation. This test check if after create product via mutation, search function working good. For strictly search functionality we have test in `saleor/core/tests/test_postgresql_search.py`. I would like to keep this test.

Reply:
If we have the test for search in particular im happy with that. I'm just not sure about the approach of testing two separate queries in one unit test which kinda violates the term and falls into pitfall of false-positive if we break the 1st mutation.